native/src/components/SubCategoryListItem.tsx

```diff
@@ -16,10 +15,10 @@ const SubCategoryTitleContainer = styled.View<{ language: string }>`
   flex-direction: ${props => contentDirection(props.language)};
 `
 
-const FlexStyledLink = styled.TouchableHighlight<{ language: string }>`
+const FlexStyledLink = styled.TouchableOpacity<{ language: string }>`
   display: flex;
   flex-direction: ${props => contentDirection(props.language)};
-  margin: 0 20px 0 60px;
+  margin: 0 0 0 24px;
```

Comment:
The 60px here is necessary to keep the visual hierarchy.
margin 0 0 0 24px:
<img width="327" alt="image" src="https://github.com/digitalfabrik/integreat-app/assets/18513943/4f2c30c3-9e80-4ad2-81e3-ec65093d9963">

margin 0 20px 0 60px: 
<img width="329" alt="image" src="https://github.com/digitalfabrik/integreat-app/assets/18513943/cce122b2-f530-405a-999c-4e83f5dcd0a8">

Reply:
I also can't reproduce this, for me it looks like this (I adjusted the margins, so there is not that much indentation anymore):
<img src="https://github.com/digitalfabrik/integreat-app/assets/29095487/0e7d3545-2d9f-43e4-8ca9-b746044e6c4f" width="200px" />